conf/modules/joint_germline.config

```
@@ -32,7 +32,7 @@ process {
 
     if (params.tools && params.tools.contains('haplotypecaller') && params.joint_germline) {
         withName: 'NFCORE_SAREK:SAREK:BAM_VARIANT_CALLING_GERMLINE_ALL:BAM_JOINT_CALLING_GERMLINE_GATK:BCFTOOLS_SORT' {
-            ext.prefix       = { "${vcf.baseName.minus("vcf")}sort" }
+            ext.prefix       = { vcf.baseName - ".vcf" + "sort" }
```

We might need to add in the `.` here. 
```suggestion
            ext.prefix       = { vcf.baseName - ".vcf" + ".sort" }
```